src/data-mgt/node/.nyc_output/processinfo/8c7a8ab0-511c-4bd2-a7c6-186399351ad3.json

```diff
@@ -0,0 +1 @@
+{"parent":null,"pid":6070,"argv":["/home/vagrant/.nvm/versions/node/v18.12.1/bin/node","/home/vagrant/AirQo-api/src/data-mgt/node/node_modules/.bin/mocha","./bin/test/ut_index.js","./bin/test/ut_kafka-consumer.js","./bin/test/ut_server.js","./config/test/ut_constants.js","./config/test/ut_database.js","./config/test/ut_log4js.js","./config/test/ut_redis.js","./controllers/test/ut_load.js","./controllers/test/ut_transform.js","./models/test/ut_channel.js","./models/test/ut_feed.js","./models/test/ut_issue.js","./models/test/ut_maintenance-logs.js","./models/test/ut_monitor.js","./models/test/ut_weather.js","./service/test/ut_monitor.js","./utils/test/ut_date.js","./utils/test/ut_errors.js","./utils/test/ut_log.js","./utils/test/ut_transform.js","--exit"],"execArgv":[],"cwd":"/home/vagrant/AirQo-api/src/data-mgt/node","time":1695296135770,"ppid":6050,"coverageFilename":"/home/vagrant/AirQo-api/src/data-mgt/node/.nyc_output/8c7a8ab0-511c-4bd2-a7c6-186399351ad3.json","externalId":"","uuid":"8c7a8ab0-511c-4bd2-a7c6-186399351ad3","files":["/home/vagrant/AirQo-api/src/data-mgt/node/bin/test/ut_index.js","/home/vagrant/AirQo-api/src/data-mgt/node/bin/test/ut_kafka-consumer.js","/home/vagrant/AirQo-api/src/data-mgt/node/bin/test/ut_server.js","/home/vagrant/AirQo-api/src/data-mgt/node/config/test/ut_constants.js","/home/vagrant/AirQo-api/src/data-mgt/node/config/test/ut_database.js","/home/vagrant/AirQo-api/src/data-mgt/node/config/test/ut_log4js.js","/home/vagrant/AirQo-api/src/data-mgt/node/config/test/ut_redis.js","/home/vagrant/AirQo-api/src/data-mgt/node/controllers/test/ut_load.js","/home/vagrant/AirQo-api/src/data-mgt/node/controllers/test/ut_transform.js","/home/vagrant/AirQo-api/src/data-mgt/node/models/test/ut_channel.js","/home/vagrant/AirQo-api/src/data-mgt/node/models/test/ut_feed.js","/home/vagrant/AirQo-api/src/data-mgt/node/models/test/ut_issue.js","/home/vagrant/AirQo-api/src/data-mgt/node/models/test/ut_maintenance-logs.js","/home/vagrant/AirQo-api/src/data-mgt/node/models/test/ut_monitor.js","/home/vagrant/AirQo-api/src/data-mgt/node/models/test/ut_weather.js","/home/vagrant/AirQo-api/src/data-mgt/node/service/test/ut_monitor.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/test/ut_date.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/date.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/log.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/test/ut_errors.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/errors.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/test/ut_log.js","/home/vagrant/AirQo-api/src/data-mgt/node/utils/test/ut_transform.js"]}
```

Comment:
Hi @Psalmz777 are we supposed to be having some of these files in this PR?
Perhaps we need to considering removing them?
I see that you already have them in your .gitignore but looks like they had already sneaked in.

In order to remove them, you can follow the following steps:

1. Remove Files Locally: `git rm --cached file_to_ignore.txt`
2. Commit the Changes: `git commit -m "Remove ignored files"`
3. Push the Changes to GitHub: `git push origin branch_name`





Thanks!

Reply:
@Baalmart lemme do that .Thanks